### ISSUE TITLE
Console errors when cloning and copying/pasting

### ISSUE THREAD

**Issue by room-roam** [spicywebau/craft-neo#697]
### Bug Description

Odd behavior and browser errors have appeared seemingly randomly. I did consider a multi site and played around with creating a new site, then reverted back to a single site. Not sure if that is the issue? I'm not sure where to begin debugging so please let me know what else I can report.

A common browser error I'm seeing:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'getId')
    at Block.js:1101:62
    at Array.find (<anonymous>)
    at s.constructor.updateMenuStates (Block.js:1101:48)
    at s.constructor.updateActionsMenu (Block.js:1008:10)
    at s.constructor._updateButtons (Input.js:556:13)
    at s.constructor.init (Input.js:179:10)
    at s.constructor.constructor (Base.js:20:15)
    at s.constructor (Base.js:32:21)
    at new s.constructor (Base.js:32:21)
    at Object.createInput (main.js:11:19)
```

### Steps to reproduce

**Blocks do not stay collapsed on entry save**
1. Collapse all blocks
2. Save entry
3. Some blocks do not collapse

https://user-images.githubusercontent.com/117244229/209363130-14fef78a-91a7-46c1-8972-a4e8a6711c0d.mp4

**Console error when cloning/copying/pasting**
1. Clone or copy a block.
2. Save entry.
3. Clone another block and browser errors.

https://user-images.githubusercontent.com/117244229/209363170-e5886a73-ffe3-47f8-85cd-da170b111c48.mp4

### Expected behaviour

1. Block collapse/uncollapsed states should persist after save.
2. Blocks should clone, copy and paste.

### Neo version

3.5.14

### Craft CMS version

4.3.5

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by ttempleton** [comment]
Thanks for reporting - should be fixed now in Neo 3.5.15.

**Comment by room-roam** [comment]
@ttempleton, thanks so much for looking into this. That solved the issue. Cheers!
